src/hacbs/components/Commits/CommitDetailsView.tsx

```diff
@@ -106,112 +157,151 @@ const CommitDetailsView: React.FC<CommitDetailsViewProps> = ({ commitName, appli
 
   if (commit) {
     return (
-      <React.Fragment>
-        <CommitsGettingStartedModal
-          shown={showGettingStarted}
-          onHide={() => setGettingStartedShown(false)}
-        />
-        <DetailsPage
-          breadcrumbs={[
-            { path: '/app-studio/applications', name: 'Applications' },
-            {
-              path: `/app-studio/applications/${applicationName}`,
-              name: applicationName,
-            },
-            {
-              path: `/app-studio/applications/${applicationName}?activeTab=commits`,
-              name: 'commits',
-            },
-            {
-              path: `/app-studio/applications/${applicationName}/commit/${commitName}`,
-              name: commitDisplayName,
-            },
-          ]}
-          title={
-            <Flex spaceItems={{ default: 'spaceItemsXs' }} className="pf-u-mt-sm pf-u-mb-lg">
-              <FlexItem>
-                <Text component="h1" data-test="details__title">
-                  {commitDisplayName}
+      <Drawer isExpanded={isExpanded} onExpand={onPanelExpand}>
+        <DrawerContent
+          panelContent={
+            sortedPLRList ? (
+              <CommitSidePanel
+                drawerRef={drawerRef}
+                isExpanded={isExpanded}
+                onPanelCloseClick={onPanelCloseClick}
+                sortedPLRList={sortedPLRList}
+                selectedPipelineRun={selectedPipelineRun}
+                setSelectedPipelineRun={setSelectedPipelineRun}
+                initialStatus={commitStatus}
+              />
+            ) : (
+              <>not found</>
+            )
+          }
+        >
+          <DrawerContentBody>
+            <CommitsGettingStartedModal
+              shown={showGettingStarted}
+              onHide={() => setGettingStartedShown(false)}
+            />
+            <DetailsPage
+              breadcrumbs={[
+                { path: '/app-studio/applications', name: 'Applications' },
+                {
+                  path: `/app-studio/applications/${applicationName}`,
+                  name: applicationName,
+                },
+                {
+                  path: `/app-studio/applications/${applicationName}?activeTab=commits`,
+                  name: 'commits',
+                },
+                {
+                  path: `/app-studio/applications/${applicationName}/commit/${commitName}`,
+                  name: commitDisplayName,
+                },
+              ]}
+              title={
+                <Text component={TextVariants.h2}>
+                  <span className="pf-u-mr-sm">
+                    <b>{commitDisplayName}</b>
+                  </span>
+                  <Button
+                    className="pf-u-pl-xs"
+                    aria-expanded={isExpanded}
+                    variant="plain"
+                    onClick={onStatusClick}
+                  >
+                    <StatusIconWithText status={commitStatus} />
```

Comment:
This component is rendering [PF badge](https://patternfly-react.surge.sh/components/badge) component, but the designs has[ PF Label ](https://patternfly-react.surge.sh/components/label)component with background colors matching the status.

current: 
<img width="449" alt="image" src="https://user-images.githubusercontent.com/9964343/192455381-4d85fdcc-411c-468b-8d17-80122c47dd96.png">

designs:
<img width="521" alt="image" src="https://user-images.githubusercontent.com/9964343/192455433-ac4e2636-d791-40d4-94c9-6f8350ba259f.png">